priv/repo/migrations/20240128194147_create_checklists.exs

```diff
@@ -0,0 +1,16 @@
+defmodule Cklist.Repo.Migrations.CreateChecklists do
+  use Ecto.Migration
+
+  def change do
+    create table(:checklists) do
+      add :title, :string
+      add :description, :string
+      add :document, :map
```

Comment:
Do I remember correctly that this should be a `jsonb` [^1]  type? If so, it's probably easiest to fix this before merging (and then re-deploy my dev database), right?

[^1]: https://www.postgresql.org/docs/current/datatype-json.html

Reply:
The type in the migration and schema is `:map`, and this is translated into a jsonb type in the database. So what you have there is correct.